sdk/src/compute_budget.rs

```diff
@@ -51,6 +55,7 @@ impl ComputeBudgetInstruction {
         borsh::to_vec(&self)
```

Comment:
I might be missing something, but if you change the config above to:

```
#[cfg(all(feature = "dev-context-only-utils", feature = "borsh"))]
```

Then you can probably remove the borsh feature flag on `dev-context-only-utils`, no?

Reply:
No, I'm suggesting to just change it for this `pack` function here. I would have commented on the exact line, but github isn't letting me, sorry